app/src/main/java/org/dhis2/utils/granularsync/SyncStatusDialog.kt

```diff
@@ -613,6 +602,12 @@ class SyncStatusDialog : BottomSheetDialogFragment(), GranularSyncContracts.View
         return getString(R.string.granular_sync_unsupported_task)
     }
 
+    override fun setLastUpdated(result: SyncDate) {
+        binding?.lastUpdated?.text = result.date?.let {
+            it.toDateSpan(requireContext())
+        } ?: "Unknown"
```

Comment:
This plain text should go to strings.xml?

Reply:
true!